src/components/schemas/Content-estimationEngine.yaml

```diff
@@ -7,3 +7,15 @@ properties:
     format: double
   priceRange:
     $ref: ./Range.yaml
+  reliability:
+    type: object
+    properties:
+      score:
+        type: integer
+        description: >-
+          Score de fiabilité calculé à partir de l'erreur moyenne et de la zone
+          géographique (0 = non fiable, 100 = très fiable)
+        format: int32
+        minimum: 0
+        maximum: 100
+        example: 78
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore. -->
​
:magic_wand: chore : Peux-tu créer un fichier dédié pour cet objet reliability nommé `EstimationResponse-reliability.yaml` ? Et du coup l'utiliser dans `Content-estimationEngine.yaml` et `EstimationResponse.yaml` ? Ca nous évitera de dupliquer de nouveau ce code à l'avenir.